server/graphql_schemas/attend/schema.py

```diff
@@ -26,20 +25,23 @@ def mutate_and_get_payload(cls, root, info, **input):
         event_id = input.get('event_id')
         event = Event.objects.get(id=event_id)
         user = info.context.user
-        google_user = GoogleUser.objects.get(app_user_id=user.id)
-        # Resolve error for users that already signified interest here
-        user_attendance = Attend(
-            user=google_user,
+        googleUser = GoogleUser.objects.get(app_user_id=user.id)
+
+        user_attendance, created = Attend.objects.get_or_create(
+            user=googleUser,
             event=event
         )
-        user_attendance.save()
+
+        if user_attendance and not created:
```

Comment:
Hi, what if this doesn't fail when the user already subscribed. In some way, I think this call should be [indempotent](https://stackoverflow.com/questions/34617702/invoking-put-request-multiple-times#34618048).

If the user already subscribed to just pass through instead of throwing an error. What do you think?

Reply:
Hi @OsayamenOsaretin 
I totally get what you are driving at and I think that's very fair, it doesn't remove anything from the usability. Let's seal it with a vote from @mbithenzomo then I will go ahead to make necessary adjustments